src/zenml/model/model.py

```diff
@@ -505,7 +505,17 @@ def _root_validator(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
     def _validate_config_in_runtime(self) -> None:
         """Validate that config doesn't conflict with runtime environment."""
-        self._get_or_create_model_version()
+        from zenml.utils.dashboard_utils import get_model_version_url
+
+        model_version = self._get_or_create_model_version()
+
+        model_version_url = get_model_version_url(model_version.id)
+        if model_version_url:
+            logger.info(f"Model version dashboard URL: {model_version_url}")
+        else:
+            logger.info(
+                "You can now view your Models in the ZenML dashboard using ZenML Cloud."
+            )
```

Comment:
Is it indeed the best place to output this? It will be called for every model version used in the pipeline. So if I set something in the pipeline and set something in 10 out of 20 steps this function will be called 11 times for each one of them (assuming they are all different, for sure).